api/python/provisioner/commands/validate_setup.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
```

Comment:
that file doesn't include any command so not a good place to locate it inside `commands`

Reply:
added this as part of pretty format of logs in `utf` format, removed now.